src/lib/fetch-document-list.js

```diff
@@ -0,0 +1,47 @@
+const { info } = require('../lib/log');
+const fs = require('../lib/sync-fs');
+const DOC_TYPES = ['district_hospital', 'health_center', 'clinic', 'person', 'user', 'user-settings', 'contact'];
+
+const jsonDocPath = (directoryPath, docID) => `${directoryPath}/${docID}.doc.json`;
+const fetchDocumentListFromDB = async (db, ids) => {
```

Comment:
```suggestion

const fetchDocumentListFromDB = async (db, ids) => {
```
space between functions